src/main/java/org/jboss/xavier/integrations/route/MainRouteBuilder.java

```diff
@@ -125,10 +128,10 @@ public void configure() throws Exception {
                 .routeId("kafka-upload-message")
                 .unmarshal().json(JsonLibrary.Jackson, FilePersistedNotification.class)
                 .filter(simple("'{{insights.service}}' == ${body.getService}"))
-                .to("direct:download-file");
+                .to("seda:download-file");
 
-        from("direct:download-file")
-                .routeId("download-file")
+        from("seda:download-file").routeId("download-file")
+                .threads(THREAD_POOL_SIZE)
```

Comment:
I took some time to understand how this works.
It seems that [SEDA component doc](https://camel.apache.org/components/latest/seda-component.html#_thread_pools) suggests to beware about using thread with seda and to prefer the `concurrentConsumers` option.
So i think we should follow this warning and move to `concurrentConsumers` option even if this means having a fixed size pool.

Reply:
concurrentConsumers is an approach that will use FIXED number of consumers
as it is said here :
"As for the difference between the two, note a *thread pool* can
increase/shrink dynamically at runtime depending on load, whereas the
number of concurrent consumers is always fixed"

Reply:
In my opinion we always going to have 1 blocking queue from the SEDA component, and with the threads we are adding another one dynamic. If we decide to increase from 5 to 20 , with the concurrentConsumers will be always having 20 consumers , but not with the threadpool